src/components/ClassBasedComponent.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
```

Comment:
you could simply use destructuring here

import React, { Component } from "react";

extends Component